frontend/src/components/product/RelatedProducts/RelatedProductsRenderer.jsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import { getRelatedSneakersById } from '@/requests/SneakersRequest';
+import RelatedProductsSection from './RelatedProductsSection';
+
+
+import '../../../styles/product/related_products.css';
+import { useRouter } from 'next/navigation';
+
+const RelatedProductsRenderer = ({ id }) => {
+	const router = useRouter();
+    const [products, setProducts] = useState(null);
+
+	for (let i = 1; i <= 5; i++) {
+        console.log(i);
+    }
+
+	useEffect(() => {
+		getRelatedSneakersById(id)
+			.then((response) => {
+				setProducts(response);
+			})
+			.catch((e) => {
+				console.error("Error fetching related sneakers:", e)
+				router.push('/');
```

Comment:
Handle this error by sending an error notification instead of changing to home.

Reply:
I will avoid redirecting, but I will not display an error message. If it does not find the related ones then it will stay in loader.

Reply:
okay